README.md

```diff
@@ -57,12 +57,14 @@ sudo make install
 
 Frontends
 ===
+  - [Weasel](https://github.com/rime/weasel): frontend for Windows
+  - [PIME](https://github.com/EasyIME/PIME): another Rime frontend for Windows
+  - [Squirrel](https://github.com/rime/squirrel): frontend for macOS
+  - [XIME](https://github.com/stackia/XIME): yet another Rime frontend for macOS
+  - [Trime](https://github.com/osfans/trime): frontend for Android
+  - [iRime](https://itunes.apple.com/cn/app/irime-shu-ru-fa/id1142623977?mt=8): frontend for iOS(**source code can't find**)
   - [fcitx-rime](https://github.com/fcitx/fcitx-rime): Fcitx frontend for Linux
```

Comment:
Shall we mark these projects " (third-party)" in their description:
PIME, XIME, fcitx-rime, Trime?